test/utils.js

```diff
@@ -783,3 +783,56 @@ export async function setupMetricsTransactions () {
 
   await MetricModel.insertMany(metrics)
 }
+
+import { connectionDefault } from '../src/config'
```

Comment:
@BMartinos Could this be moved up with the rest of the imports for consistency - or was there a reason for having it here?

Reply:
This should be added to the top where all the imports are done. This was part of quick development getting things in place and needs cleanup

Reply:
Done in 737fc21